examples/contractCreation/modest.act

```diff
@@ -0,0 +1,18 @@
+// a modest example specifying the creation of a new contract,
+// in this case the contract subject at question.
+// Speciying the creation of a contract serves multiple purposes:
+// from a formal semantics perspective, it defines the base case
+// for the inductive system that is a smart contract. From a syntactic
+// perspective, it defines the storage layout of the contract, by explicit
+// typing of storage variables.
+
+behaviour creation of Modest
+interface constructor()
+
+creates storage
+    // storage variables must be given an explicit initial value
+    // if they are not assigned a value in the constructor,
+    // this is 0. TODO (is this annoying?)
```

Comment:
Maybe we should be explicit and require one and only one assignment?

Reply:
yup. One assignment only!